Changelog.md

```diff
@@ -0,0 +1,30 @@
+# CtTSO-ICA-To-PierianDx Changelog
+
+Changes in this log refer only to changes that make it to the 'main' branch. and
+
+For changes in deployment, please see the [deployment changelog](deploy/cttso-ica-to-pieriandx-cdk/Changelog.md) 
+
+## 2022-08-30  
+
+> Author: Alexis Lucattini
+> Email: [Alexis.Lucattini@umccr.org](mailto:alexis.lucattini@umccr.org)
+
+### Summary 
+
+* Wrapped portal Run ID name collector in try-exception clause
```

Comment:
Should this not fail if the portal is missing key information for a given sample? Just making sure I understand the motivation here - did we have valid samples waiting for submission to PierianDX that the portal did not have the relevant info for?

Reply:
Was added in since if we try run this in the dev environment, it could fail, and would rather just get to the end of the submission and throw a warning than fail the job entirely since it's likely I'm trying to test other components.